rasa/nlu/run.py

```diff
@@ -1,22 +1,32 @@
+import asyncio
 import json
 import logging
 
 from rasa.cli.utils import print_success
 
 from rasa.nlu.model import Interpreter
+from rasa.core.interpreter import RegexInterpreter, INTENT_MESSAGE_PREFIX
 
 logger = logging.getLogger(__name__)
 
 
 def run_cmdline(model_path, component_builder=None):
     interpreter = Interpreter.load(model_path, component_builder)
+    regex_interpreter = RegexInterpreter()
 
     print_success("NLU model loaded. Type a message and press enter to parse it.")
     while True:
-        text = input().strip()
-        r = interpreter.parse(text)
-        print (json.dumps(r, indent=2))
         print_success("Next message:")
+        message = input()
+        if message.startswith(INTENT_MESSAGE_PREFIX):
+            text = message.rstrip()
```

Comment:
why do we not use `strip` for both cases?

Reply:
I think I realized this mistake right after requesting the review and leaving the office 😅fixes